.codeclimate.yml

```diff
@@ -0,0 +1,42 @@
+version: "2"         # required to adjust maintainability checks
+checks:
+  argument-count:
+    config:
+      threshold: 4
+  complex-logic:
+    config:
+      threshold: 4
+  file-lines:
+    config:
+      threshold: 250
+  method-complexity:
+    config:
+      threshold: 5
+  method-count:
+    config:
+      threshold: 20
+  method-lines:
+    config:
+      threshold: 25
+  nested-control-flow:
+    config:
+      threshold: 4
+  return-statements:
+    config:
+      threshold: 4
+  similar-code:
+    config:
+      threshold: # language-specific defaults. an override will affect all languages.
+  identical-code:
+    config:
+      threshold: # language-specific defaults. an override will affect all languages.
+plugins:
+  bandit:
+    enabled: true
+  pep8:
+    enabled: false
+  radon:
+    enabled: true
+  sonar-python:
+    enabled: true
```

Comment:
The [docs](https://docs.codeclimate.com/docs/sonar-python) show a `minimum_severity` argument that we may want to employ so as to not generate too much noise.